src/components/Footer/Footer.js

```diff
@@ -1,7 +1,90 @@
+import { useNavigate } from 'react-router-dom';
 import './Footer.scss';
 
 function Footer() {
-  return;
+  const navigate = useNavigate();
+
+  if ((window.location.pathname === '/signup', '/')) return null;
+
+  const goToLogin = () => {
+    navigate('/signup');
+  };
+
+  return (
+    <div>
```

Comment:
최상위 태그의 className은 컴포넌트랑 같은 이름으로 해주세요.
```suggestion
    <div className="footer">
```

Reply:
네 반영하였습니다!